src/components/Feedback/constants.js

```diff
@@ -0,0 +1,2 @@
+export const FeedbackTypes = ['Review', 'Bug', 'Idea'];
+export const primaryColor = '#00262b';
```

Comment:
do we have this color code available in paragon? can we use that from there?

Reply:
We can only pass the color CSS value such as HEX, RGB, and paragon provides string classes such as `text-primary-500` which is not compatible with it. I tried this solution too.